test/expected/size_utils.out

```diff
@@ -179,8 +209,14 @@ SELECT * FROM hypertable_detailed_size(NULL);
 -------------+-------------+-------------+-------------+-----------
 (0 rows)
 
-SELECT * FROM indexes_relation_size(NULL);
- index_name | total_bytes 
-------------+-------------
-(0 rows)
+SELECT * FROM hypertable_index_size(NULL);
+ hypertable_index_size 
+-----------------------
+                      
+(1 row)
 
+-- tests with tables that are not hypertables
+CREATE TABLE regtab( a integer, b integer);
+CREATE INDEX regtab_idx ON regtab( a);
+SELECT * FROM hypertable_index_size('regtab_idx');
+ERROR:  query returned no rows
```

Comment:
Strange that this is an error. "No rows" are usually not considered an error.

Reply:
That's because we don't have any explicit code that handles non hypertables.